h2o-core/src/main/java/hex/ParallelModelBuilder.java

```diff
@@ -55,21 +55,25 @@ public void run(final Collection<ModelBuilder> modelBuilders) {
 
     @Override
     public void onModelSuccess(Model model) {
-      try {
-        _callback.onBuildSuccess(model, ParallelModelBuilder.this);
-      } finally {
-        _modelInProgressCounter.decrementAndGet();
+      if (! model._parms._is_cv_model) {
```

Comment:
Maybe this logic could be incorporated one level lower - in the ModelBuilder itself - do not report CV models. Not sure the side ffects are right - the clients should probably decide. So on the other hand, this is the perfect "sweet spot". :+1: 

Thank you @michalkurka .

I can add some JUnit tests if @sebhrusen thinks this solution is allright and that's it.